docs/midi.md

```diff
@@ -0,0 +1,18 @@
+# MIDI
+The MIDI module adds keymap entries for sending MIDI data streams. It will require adding the `adafruit_midi` library from the [Adafruit CircuitPython Bundle](https://circuitpython.org/libraries) to your device's folder.  
+Add it to your keyboard's modules list with:
+
+```python
+from kmk.modules.midi import MidiKeys
+keyboard.modules.append(MidiKeys())
+```
+## Keycodes
+
+|Key         |Description                                                                  |
+|-----------------|------------------------------------------------------------------------|
+|`KC.MIDI_CC()`      |Sends a ControlChange message; accepts two integer arguments of `0`-`15`(controller number) then `0`-`127`(control value)                                           |
+|`KC.MIDI_NOTE()`      |Sends a Note message with both 'On' and 'Off' segments; accepts two integer arguments of `0`-`127`(note number) and `0`-`127`(velocity)            |
+|`KC.MIDI_PB()` |Sends a Pitch Wheel message; accepts a single integer argument of `0`-`16383`, centered on `8192`                                 |
+|`KC.MIDI_PC()`  |Sends a Program Change message; accepts a single integer argument of `0`-`127`(program number)               |
+|`KC.MIDI_START()`      |Sends a Start message; accepts no arguments          |
+|`KC.MIDI_STOP()`      |Sends a Stop message; accepts no arguments                     |
```

Comment:
* Add arguments to the key codes, like: `KC.MIDI_NOTE(note, velo)`
* table formating (raw text) is a bit off
* MIDI channel selection is missing

I don't think you have to describe the MIDI protocol in detail in the key code table, I'm sure everyone who needs MIDI will be familiar with value ranges and that they have to be integers.

Reply:
- Will be added, forgot to go back to it
 - Yeah idk how I let it get that bad
 - Is that a command or a parameter? I was just going off the keys laid out in `midi.py`

Reply:
The key codes have the same signature as the adafruit_midi messages: `KC.MIDI_NOTE(note, velocity, channel=0)`, where `channel` is a named optional argument. No need to list that in the table, just mention it in a paragraph below.